minor fixes, updates (#354)
slsa-framework/slsa-github-generator: internal/builders/generic/README.md

$ sha256sum artifact1 artifact2 ... | base64 -w0
```

> This workflow expects the base64 encoded value for the subjects to decode to a string conforming to the expected output of the above command. Specifically, the decoded output is expected to be comprised of a hash value followed by a space followed by the artifact name.

Reviewer: Technically the above command outputs the base64 value so the value of `base64-subjects` wouldn't wouldn't decode to that. It would decode to the value of the `sha256sum artifact1 artifact2 ...` part. Rather than "above command". I'd just say sha256sum command.

Also consider referring to the input name directly so users know exactly what we're talking about.
e.g. "This workflow expects the value of `base64-subjects to ..."
```suggestion
This workflow expects the `base64-subjects` input to decode to a string conforming to the expected output of the `sha256sum` command. Specifically, the decoded output is expected to be comprised of a hash value followed by a space followed by the artifact name.
```

Reviewer: Additionally, consider adding this as a comment in the example yaml below.
```yaml
with:
  # This workflow expects the value of base64-subjects to decode to a string conforming to the
  # expected output of the sha256sum command...
  base64-subjects: ...

### Workflow Inputs

> The [builder workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) accepts the following inputs:

Reviewer: Let's change "builder workflow" to "generic workflow" here since that's the wording we're using elsewhere.
```suggestion
The [generic workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) accepts the following inputs:
```

| Name              | Required | Description                                                                                                                        |
| ----------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
| `base64-subjects` | yes      | Artifact(s) for which to generate provenance, formatted the same as the output of sha256sum (SHA256 NAME\n[...]) and base64 encoded. The encoded value should decode to, for example: `90f3f7d6c862883ab9d856563a81ea6466eb1123b55bff11198b4ed0030cac86  foo.zip` |

### Workflow Outputs

> The [builder workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) produces the following outputs:

Reviewer: ```suggestion
The [generic workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) produces the following outputs:
```
